larq_compute_engine/tflite/python/interpreter.py

```diff
@@ -46,16 +19,12 @@ class Interpreter:
         interpreter.predict(input_data, verbose=1)
         ```
 
+    See the base class `InterpreterBase` for the full interface.
+
     # Arguments
         flatbuffer_model: A serialized Larq Compute Engine model in the flatbuffer format.
         num_threads: The number of threads used by the interpreter.
         use_reference_bconv: When True, uses the reference implementation of LceBconv2d.
-
-    # Attributes
-        input_types: Returns a list of input types.
-        input_shapes: Returns a list of input shapes.
-        output_types: Returns a list of output types.
-        output_shapes: Returns a list of output shapes.
```

Comment:
Can we add the Attributes, here again since they appear in [our docs](https://docs.larq.dev/compute-engine/api/python/#interpreter)?

Reply:
Won't the docs pick it up from the base class?

Reply:
Unfortunately I don't think so, it just parses the markdown.

Reply:
I've moved it: https://github.com/larq/compute-engine/pull/677